### DISCUSSION TITLE
plugin-survey css and cognition.run help

### DISCUSSION THREAD

**Discussion by reillyinnes** [jspsych/jsPsych#2982] [Q&A]
Hi all,

I'm creating a survey in cognition.run using the survey plugin and was after some help with the appearance. All I want to do is change the style to be the same from the survey to the regular jsPsych css (specifically the appearance of the buttons to be the same as the button-response plugins and some alignment changes). 

I found the survey.css, updated it using the jspsych.css code and uploaded to cognition.run, but there is no change to appearance. This is (hopefully) probably a simple fix but I cannot seem to get it to work. Any help (or even an example similar) would be greatly appreciated!

(btw, thanks again to Josh and team for this amazing bit of coding brilliance!)

**Comment by PollyBrehautj** [comment]
run using the survey plugin, but are having trouble getting the changes to show up. Have you tried clearing your browser cache to make sure you're seeing the updated CSS? That could be a simple solution.

If that doesn't work, you might want to double-check that you've correctly updated the survey.css file with the jspsych.css code. It's possible that there's an error in the updated code that's preventing the changes from taking effect.

If you're still having trouble, you could try reaching out to the cognition.run support team for assistance or posting on the jsPsych forum to see if anyone else has had a similar issue. Good luck!

**Reply by reillyinnes** [reply]
Thanks for this! 

Turns out the .css file I uploaded was not formatted correctly (tip for the young players - use a decent text editor). I also had trouble tracking down which element I actually wanted to style, but eventually got there through some back and forth between inspecting elements and tracking down the original survey.css source code (which can be found in the source code, not in the compiled version).

**Reply by GABowers** [reply]
I'm trying to do the same thing - give the survey buttons an appearance and position similar to the rest of the plugins. Forgive me for being gauche, but would you be willing to share this?
